public/js/main.js

```diff
@@ -44,6 +44,13 @@ function initializeClock(id, endtime) {
     var timeinterval = setInterval(updateClock, 1000);
 }
 
+function CheckDate(){
+    var today = new Date();
+    if (today.getHours() <= 12 || today.getHours > 17){
+        window.location = 'http://treasurehunt.cshub.ca/Closed';
+    }
+
+}
 //var deadline = new Date(Date.parse(new Date()) + 15 * 24 * 60 * 60 * 1000);
```

Comment:
This shouldn't be commented out but rather set to finishing time,date of the treasure hunt.

Reply:
The end time is higher up in the webpage it seems to be there twice